cfme/services/requests.py

```diff
@@ -175,7 +175,7 @@ def wait_for_request(cells, partial_check=False):
         # Request not found at all, can't continue
         return False
 
-    assert row.status.text != 'Error'
+    # assert row.status.text != 'Error'
```

Comment:
This should probably just get deleted, without a comment. Someone (me, if nobody else beats me to it) needs to look at tests that call this method and make sure we don't have some that are expected an AssertionError to be raised here.